docs/cn/advanced/Tiered-Locality.md

```diff
@@ -22,7 +22,7 @@ priority: 1
 alluxio.locality.[tiername]=...
 ```
 
-请参阅[配置项设置](Configuration-Settings.html)页面以获取详细信息来设置配置属性。
+请参阅[配置项设置]({{ '/cn/operation/Configuration.html' | relativize_url }})页面以获取详细信息来设置配置属性。
```

Comment:
Change to `/en/reference/Properties-List.html`?

Reply:
Same, configuration